docs/Documentation/Slurm/batch_jobs.md

```diff
@@ -65,7 +65,7 @@ You may use these environment variables in your sbatch scripts to help control o
 | ---------------- | ----------------- | ------------------------ |
 | `$LOCAL_SCRATCH` | Absolute directory path for local-only disk space per node. This should always be /tmp/scratch for compute nodes with local disk.| `/tmp/scratch`|
 | `$TMPDIR` | Path for temporary directory for scratch space. Uses local storage on compute nodes with local disk, and RAM on those without. | `/tmp/scratch/<JOBID>` (default value on Kestrel)|
-| `$SLURM_CLUSTER_NAME` | The cluster name as per the master configuration in Slurm. Identical to `$NREL_CLUSTER`. | `kestrel`, `swift`|
+| `$SLURM_CLUSTER_NAME` | The cluster name as per the master configuration in Slurm. Identical to `$NLR_CLUSTER`. | `kestrel`, `swift`|
```

Comment:
This variable does not exist yet. Do you think it is alright to adjust this variable early?

Reply:
Good catch thank you, I'll change that back